core-services/prow/02_config/openshift/cluster-api/_prowconfig.yaml

```diff
@@ -72,6 +72,8 @@ tide:
     - main
     - master
     - openshift-4.1
+    - openshift-4.10
+    - openshift-4.11
```

Comment:
Ok this config file was for some reason wrong. This update to the complement query is fine, but the `staff-eng-approved` one needs to have `4.10`  branches and the `cherry-pick-aproved` one needs to have 4.9 and 4.10 ones added.